substrate/client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -198,6 +218,8 @@ where
 			};
 			debug!(target: LOG_TARGET, "[follow][id={:?}] Subscription accepted", sub_id);
 
+			let _registered_token = reserved_token.register(sub_id.clone());
```

Comment:
doesn't return anything?!

```suggestion
			reserved_token.register(sub_id.clone());
```